File: java/05_functions.java
public class Functions {
    static String greet(String name) {
        return "hello " + name;
    }

    static int add(int a, int b) {
        return a + b;
    }

    public static void main(String[] args) {
        System.out.println(greet("ulas"));
        System.out.println(add(3, 5));
    }
}

File: python/05_functions.py
# 05 functions

def greet(name):
    return f"hello {name}"

print(greet("ulas"))

def add(a, b):
    return a + b

print(add(3, 5))

# default argument
def say(msg="hi"):
    print(msg)

say()
say("yo")
